includes/EditPage.php

```diff
@@ -2621,26 +2621,26 @@ protected function wasDeletedSinceLastEdit() {
 
 	protected function getLastDelete() {
 		$dbr = wfGetDB( DB_SLAVE );
-		$data = $dbr->selectRow(
-			array( 'logging', 'user' ),
-			array( 'log_type',
-				   'log_action',
-				   'log_timestamp',
-				   'log_user',
-				   'log_namespace',
-				   'log_title',
-				   'log_comment',
-				   'log_params',
-				   'log_deleted',
-				   'user_name' ),
-			array( 'log_namespace' => $this->mTitle->getNamespace(),
-				   'log_title' => $this->mTitle->getDBkey(),
-				   'log_type' => 'delete',
-				   'log_action' => 'delete',
-				   'user_id=log_user' ),
-			__METHOD__,
-			array( 'LIMIT' => 1, 'ORDER BY' => 'log_timestamp DESC' )
-		);
+		$data = $dbr->selectRow( [ 'logging' ], [
+			'log_type',
+			'log_action',
+			'log_timestamp',
+			'log_user',
+			'log_namespace',
+			'log_title',
+			'log_comment',
+			'log_params',
+			'log_deleted',
+		], [
+			'log_namespace' => $this->mTitle->getNamespace(),
+			'log_title' => $this->mTitle->getDBkey(),
+			'log_type' => 'delete',
+			'log_action' => 'delete',
+		], __METHOD__, [
+			'LIMIT' => 1,
+			'ORDER BY' => 'log_timestamp DESC'
+		] );
+
```

Comment:
`user_name` is no longer returned by this query. Is it calculated later on?

Reply:
Please mark core changes with `SUS-...` ticket ID. We do still perform backports from MediaWiki upstream. Such comments help a lot.

Reply:
No it isn't calculated. This method is used only in [\EditPage::wasDeletedSinceLastEdit](https://github.com/Wikia/app/blob/72b0a80ae843baa2f534a49622242805063a0bf5/includes/EditPage.php#L2610) and there is no need to get any of user data in this case

Reply:
But the user name of last delete is used on https://github.com/Wikia/app/blob/72b0a80ae843baa2f534a49622242805063a0bf5/includes/EditPage.php#L1841 .

Reply:
Thanks! I didn't catch that one